go.mod

```diff
@@ -10,6 +10,7 @@ require (
 	github.com/grpc-ecosystem/go-grpc-middleware v1.0.1-0.20190118093823-f849b5445de4
 	github.com/grpc-ecosystem/go-grpc-prometheus v1.2.0
 	github.com/hashicorp/vault/api v1.0.5-0.20200902155336-f9d5ce5a171a
+	github.com/kube-storage/spec v0.0.0-20210215170332-968ad288fb6d
```

Comment:
is there a pre-release that can be tagged and used? that way we can easily see what changes are made in the upstream project in case a future rebase is needed

Reply:
Not yet. as we are sticking to a particular commit should be fine. but we will have a release before the cephcsi release so that we can stick to the released version.